packages/client3/components/design-system/logo/logo.tsx

```diff
@@ -1,16 +1,56 @@
+import Image from "next/image";
+
+import FreeArtistsFullLogo from "/public/FAD-logo-full.png";
+
 import freeArtistsLogoWhiteBgPng from "./FAD-logo-white-bg.png";
 import freeArtistsLogoPng from "./FAD-logo.png";
 import FreeArtistsLogoSvg from "./FAD-logo.svg";
 
-export const freeArtistsLogoPngUrl = freeArtistsLogoPng.src;
-export const freeArtistsLogoWhiteBgPngUrl = freeArtistsLogoWhiteBgPng.src;
+export const freeArtistsLogoPngUrl = "/images/fa-logo.png";
+export const freeArtistsLogoWhiteBgPngUrl = "/public/fa-logo-white-bg.png";
+export const freeArtistsFullLogoPngFull = FreeArtistsFullLogo.src;
 
 interface FreeArtistsLogoProps {
   width?: number;
   height?: number;
   className?: string;
+  type?: FreeArtistsLogoType;
+  alt?: string;
+  src?: string;
+  storyBookMode?: string;
+}
+
+enum FreeArtistsLogoType {
+  pngWhiteBg,
+  png,
+  pngFull,
 }
 
 export function FreeArtistsLogo(props: FreeArtistsLogoProps) {
   return <FreeArtistsLogoSvg {...props} />;
 }
+
+export function FreeArtistsLogoFull(props: FreeArtistsLogoProps) {
+  const src = props.src;
+  const freeArtistsFull = "http://localhost:3001/FAD-logo-full.png";
+
+  if (props.storyBookMode) {
```

Comment:
I think this was added to prevent the issue with Next Image in the storybook. But then this won't be an ideal approach as storybook could be considered the source of truth when it comes to components but here the components rendered are different(even though minor difference) when the storybook is used or not.

Had a look at the storybook issue with the next image and found something that could be helpful. ([link](https://storybook.js.org/blog/get-started-with-storybook-and-next-js/)).